images/bootstrap/Makefile

```diff
@@ -18,7 +18,8 @@ TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 all: build
 
 build:
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./barnacle/barnacle ./barnacle/
+	bazel build //images/bootstrap/barnacle:barnacle
```

Comment:
add a comment?

Reply:
what should I comment?

Reply:
discussed offline, leaving as-is for now.